.github/workflows/python-packaging.yml

```diff
@@ -24,11 +36,15 @@ jobs:
       run: python setup.py sdist
       working-directory: ./apis/python
     - name: Upload sdist artifact
+      # One might ask "Upload" -- to where? This means you can go to 
```

Comment:
Tiny, tiny nit:  this is well covered by the docs for the action.  Not sure recapitulating it here is helpful (and if it is, why is it not done for all of the other canned actions)?

Reply:
I assert it would have been helpful had it been here -- either that, or
https://github.com/single-cell-data/TileDB-SOMA/wiki/PyPI-packaging-WIP
as an act of kindness to our co-workers